### PR TITLE
DataParser changes in way to read version from document

### PR DESCRIPTION
**Pull request by ZaidMirza**
.first().ownText() will not work as Version info is not at first element in document. I have changed and updated it

### CODE REVIEW

File: gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java [hunk 1]
```diff
@@ -18,9 +19,18 @@
     private static final String DIV_DESCRIPTION = "div[itemprop=description]";
 
     public Version parse(Document document, final String currentVersion, final String url) {
-        final String newVersion = document.select(DIV_VERSION)
-                .first()
-                .ownText();
+         String newVersion ="";
+        if (document != null) {
+            Elements element = document.getElementsContainingOwnText("Current Version");
```
Comment (robohorse) [inline gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java:24]: This code will work on the app html page ENGLISH version only( But, that is good idea - we can make 2 requests to make this library alive again!

Comment (robohorse) [reply gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java:24]: i will do some changes later and publish new version! Thank you!

Comment (ZaidMirza) [reply gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java:24]: Please update it as soon as possible.

Comment (robohorse) [reply gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java:24]: fixed in 1.6.0

Comment (ZaidMirza) [reply gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java:24]: Thanks,

**Pull request merged**